samples/LocalizationSample/Startup.cs

```diff
@@ -23,31 +22,19 @@ public void ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app, IStringLocalizer<Startup> SR)
         {
-            var supportedCultures = new List<CultureInfo>
-            {
-                new CultureInfo("en-US"),
-                new CultureInfo("en-AU"),
-                new CultureInfo("en-GB"),
-                new CultureInfo("es-ES"),
-                new CultureInfo("ja-JP"),
-                new CultureInfo("fr-FR"),
-                new CultureInfo("zh"),
-                new CultureInfo("zh-CN")
-            };
-            var options = new RequestLocalizationOptions
-            {
-                DefaultRequestCulture = new RequestCulture("en-US"),
-                SupportedCultures = supportedCultures,
-                SupportedUICultures = supportedCultures
-            };
-            // Optionally create an app-specific provider with just a delegate, e.g. look up user preference from DB.
-            // Inserting it as position 0 ensures it has priority over any of the default providers.
-            //options.RequestCultureProviders.Insert(0, new CustomRequestCultureProvider(async context =>
-            //{
-
-            //}));
+            var supportedCultures = new [] { "en-US", "en-AU", "en-GB", "es-ES", "ja-JP", "fr-FR", "zh", "zh-CN" };
+            app.UseRequestLocalization(options =>
+                options
+                    .AddSupportedCultures(supportedCultures)
+                    .AddSupportedUICultures(supportedCultures)
+                    .SetDefaultCulture(supportedCultures[0])
```

Comment:
Can we not take dependency through an index, as that will break if someone will modify the list in the future. Let's keep the 'en-US' as the default explicitly.

Reply:
I don't think this will break while all the cultures are supported